utils/record_video.py

```diff
@@ -3,27 +3,71 @@
 
 import gym
 import numpy as np
-from stable_baselines3.common.vec_env import VecVideoRecorder, VecFrameStack, VecNormalize
+from stable_baselines3.common.vec_env import (
+    VecVideoRecorder,
+    VecFrameStack,
+    VecNormalize)
 
-from .utils import ALGOS, create_test_env, get_saved_hyperparams, get_latest_run_id
+from .utils import (
+    ALGOS,
+    create_test_env,
+    get_saved_hyperparams,
+    get_latest_run_id)
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--env', help='environment ID', type=str, default='CartPole-v1')
-parser.add_argument('-f', '--folder', help='Log folder', type=str, default='rl-trained-agents')
-parser.add_argument('-o', '--output-folder', help='Output folder', type=str, default='logs/videos/')
-parser.add_argument('--algo', help='RL Algorithm', default='ppo',
-                    type=str, required=False, choices=list(ALGOS.keys()))
-parser.add_argument('-n', '--n-timesteps', help='number of timesteps', default=1000,
-                    type=int)
-parser.add_argument('--n-envs', help='number of environments', default=1,
-                    type=int)
-parser.add_argument('--deterministic', action='store_true', default=False,
-                    help='Use deterministic actions')
-parser.add_argument('--seed', help='Random generator seed', type=int, default=0)
-parser.add_argument('--no-render', action='store_true', default=False,
-                    help='Do not render the environment (useful for tests)')
-parser.add_argument('--exp-id', help='Experiment ID (default: 0: latest, -1: no exp folder)', default=0,
-                    type=int)
+parser.add_argument(
```

Comment:
same question here, the line was too long?

Reply:
Yup, all the parser arguments are too long.